packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -0,0 +1,47 @@
+import {Component, Element, h} from '@stencil/core';
+import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rounded.svg';
+import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
+
+@Component({
+  tag: 'atomic-segmented-facet-scrollable',
+  styleUrl: 'atomic-segmented-facet-scrollable.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacetScrollable {
+  @Element() private host!: HTMLElement;
+  // @State() children: Array<any> = [];
+
+  connectedCallback() {
+    // this.children = Array.from(this.host.children);
+    // console.log(this.children);
+
+    // const shadowRoot =
+    //   this.host.shadowRoot === null
+    //     ? this.host.attachShadow({mode: 'open'})
+    //     : this.host.shadowRoot;
+
+    console.log(this.host.shadowRoot);
+  }
```

Comment:
won't need his in the end I guess? same for the children state

Reply:
yup, I'll remove it